lib/topological_inventory/providers/common/save_inventory/saver.rb

```diff
@@ -117,6 +117,10 @@ def build_new_inventory(inventory)
             new_inventory[:collections]             = []
             new_inventory
           end
+
+          def payload_max_size
+            ENV['KAFKA_MESSAGE_MAX_BYTES']&.to_i || 1_000_000
```

Comment:
With this new method I suggest moving the subtraction of `KAFKA_RESERVED_HEADER_SIZE` here instead of doing it in the `initialize` method.

Reply:
Agree.
Also KAFKA_RESERVED_HEADER_SIZE = 200 seems too small. Not only ruby-kafka adds some headers, manageiq-messaging also adds its own headers. How able increasing it to 512?